src/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework/Microsoft.CommonLanguageServerProtocol.Framework.Shared.projitems

```diff
@@ -19,7 +19,6 @@
     <Compile Include="$(MSBuildThisFileDirectory)Handlers\InitializedHandler.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Handlers\InitializeHandler.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)IInitializeManager.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)ILifeCycleManager.cs" />
```

Comment:
I don't think the definition here can be removed - we have other consumers of clasp (as a source package) like Razor

We can modify the interface (since its a source package), but the functionality must still be available in clasp

Reply:
Fixed as discussed in chat and tested